configure

```diff
@@ -1323,6 +1327,9 @@ Optional Packages:
   --with-reports-hostname=HOSTNAME
                           Use HOSTNAME as hostname for statistics collection
                           and update checks
+  --without-columnar-tableam
+                          compile with table access methods for columnar
```

Comment:
is this option useful / does it work? wouldn't we still try to CREATE ACCESS METHOD in the SQL script?

Reply:
True that.

Maybe its better to infer the define from the pgversion without any configure options. I started with the option to get running quick and be able to configure without for pg11. Then I found that it was better to auto sense the pgversion, but kept the option here.

Will remove.